protocol/communities/community.go

```diff
@@ -1995,31 +1994,14 @@ func (o *Community) AddMemberWithRevealedAccounts(dbRequest *RequestToJoin, role
 	defer o.mutex.Unlock()
 
 	isControlNode := o.IsControlNode()
-	allowedToSendEvents := o.HasPermissionToSendCommunityEvents()
 
-	if !isControlNode && !allowedToSendEvents {
+	if !isControlNode {
```

Comment:
Admins should not make changes to members directly, so only control node is allowed to do stuff here.
Also, we want to move away from attaching revealed addresses to members in the community description (as this is public metadata).

Next step is to remove usage of this function entirely.